spec/controllers/albums_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe AlbumsController, type: :controller do
+  let(:good_params) do
+    {
+      album: {
+        title: "Hello, World!",
+        creator: "Jennie"
+      }
+    }
+  end
+  let(:bad_params) do
+    {
+      album: {
+        title: "Hello, World!"
+      }
+    }
+  end
+  let(:update_params) do
+    {
+      id: new_medium.id,
+      album: {
+        description: "Great product!" }
+    }
+  end
+  let(:bad_update_params) do
+    {
+      id: new_medium.id,
+      album: {
+        title: "" }
+    }
+  end
+  let(:model) do
```

Comment:
Shorter `let`s like these can go on one line, like so: `let(:model) {  "Album".constantize }`. I get it if it's easier for you to read them this way, but a lot of style guides will suggest that you minimize the whitespace. The other `let`s are not so complicated that they can't also go on one line, but the whitespace makes sense there.